Runtime/ui/renderer/cmdbufferCanvas/rendering/canvas_shader.cs

```diff
@@ -62,7 +63,10 @@ static class MaterialProps {
                 mat.SetInt(_dstBlend, (int) UnityEngine.Rendering.BlendMode.Zero);
             }
             else {
-                Debug.LogWarning("Not supported BlendMode: " + op + ". Defaults to srcOver");
+                D.assert(() => {
+                    Debug.LogWarning("Not supported BlendMode: " + op + ". Defaults to srcOver");
```

Comment:
why fix here ? a Debug.LogWarning nested in D.assert appears abnormal

Reply:
So that the warning information appears only when debug is turned on.